framework/src/application/node/synchronizer/utils.js

```diff
@@ -40,8 +40,8 @@ const restoreBlocks = async (blocksModule, processorModule, tx = null) => {
 	return true;
 };
 
-const clearBlocksTempTable = storageModule =>
-	storageModule.entities.TempBlock.truncate();
+const clearBlocksTempTable = blocksModule =>
```

Comment:
can we remove this and directly use `blocksModule.dataAccess.clearTempBlocks`? it's not reused as well

Reply:
That was the original function so I will keep it for now.